backend/api/models/Message.py

```diff
@@ -9,22 +9,29 @@ class Message(Mixin, db.Model):
 
     __tablename__ = "message"
 
+    id = db.Column(db.String, unique=True, primary_key=True)
     pm_id = db.Column(db.String, db.ForeignKey("portfolio_manager.id"))
     fp_id = db.Column(db.String, db.ForeignKey("field_partner.id"))
-    to_fp = db.Column(db.Boolean)  # true if send to fp; false if send to pm
+    # true if send to fp; false if send to pm
+    to_fp = db.Column(db.Boolean)
 
-    doc_id = db.Column(db.Integer, unique=True, primary_key=True)
-    status = db.Column(db.String, unique=True)
-    comment = db.Column(db.String, nullable=True)
+    # These are all nullable depending on the type of notification
+    doc_id = db.Column(db.Integer, db.ForeignKey("documents.id"), nullable=True)
+    status = db.Column(db.String, nullable=True)
+    description = db.Column(db.String, nullable=True)
+    time = db.Column(db.String)
```

Comment:
would a datetime work better or was this chosen to avoid conversion issues?

Reply:
i think the time thing was there before i started messing with the notifications, but it was inside a "comment" column- i just copied it over